### ISSUE TITLE
Aurelia-CLI and sub-dependencies

### ISSUE THREAD

**Issue by seagullmouse** [braintree/braintree-web#360]
### General information

* SDK version: 3.32.1
* Environment: Sandbox
* Browser and OS: Mac OS 10.13.4

### Issue description

I am trying to use https://www.npmjs.com/package/braintree-web with Aurelia (using the aurelia-cli and RequireJS). I am stuck trying to get all the many dependencies to resolve.

To use 3rd party library in Aurelia the library must be defined in the aurelia.json file. 

If I add "braintree-web" in that file then aurelia complains that "braintree-web" it requires the modules "american-express", "apple-pay" etc etc. 

If I manually create the "american-express", "apple-pay" dependencies then each one also refers to "braintree-web/lib", and a bunch of other sub-directory dependencies.

In short I can't get the "braintree-web" module to load because I have to manually build all sub-dependencies and its too complex to get working.

As I state above, I am using requireJS, should these dependencies all resolve correctly?

Any ideas as to how I can get this working?

Thanks

**Comment by jackellenberger** [comment]
Hey @seagullmouse, we're not terribly familiar with Aurelia, so we won't be of too much help - our suggestion would just be to load braintree-web through our CDN hosted link. It's possible that there is a way to use the npm packaged version, but [the RequireJS documentation](http://requirejs.org/docs/node.html#2) will be the best place to look for that.

**Comment by seagullmouse** [comment]
Thanks, I'm trying to find a way to use the bundled code.
